inventory/aws.cf

```diff
@@ -0,0 +1,21 @@
+bundle common inventory_aws
+# @brief inventory AWS EC2 instances
+#
+# Provides:
+#  ec2_instance class based on Amazon markers in dmidecode's system-uuid, bios-version or bios-vendor
+{
+  classes:
+    any::
+      "ec2_instance"
+        comment => "See http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/identify_ec2_instances.html",
+        or => {
+          regcmp("^[eE][cC]2.*", "$(cfe_autorun_inventory_dmidecode.dmi[system-uuid])"),
```

Comment:
I think that this may be skipped if the variable is not defined. Probably I will separate these into 3 individual promises for resilience.